src/bic/model-interaction.js

```diff
@@ -224,27 +258,8 @@ The argument change event.
             reject();
           }
         } else {
-          model.set({interactionList: _.map(_.filter(app.interactions.models, function (value) {
-            return value.id !== window.BMP.BIC.siteVars.answerSpace.toLowerCase() && value.get('display') !== 'hide' && (!value.has('tags') || value.has('tags') && value.get('tags').length === 0 || _.filter(value.get('tags'), function (element) {
```

Comment:
I believe this PR actually deletes those lines, anyway. There is another spot though where this might still apply...